backend/service/topology/cache.go

```diff
@@ -31,15 +31,9 @@ func (c *client) acquireTopologyCacheLock(ctx context.Context) {
 	}
 	defer conn.Close()
 
-	go func() {
-		<-ctx.Done()
-		ticker.Stop()
-		c.unlockAdvisoryLock(context.Background(), conn, advisoryLockId)
-	}()
-
 	// Infinitely try to acquire the advisory lock
 	// Once the lock is acquired we start caching, this is a blocking operation
-	for ; true; <-ticker.C {
+	for {
```

Comment:
sorry if I'm missing it somewhere else in the code, but do you need to create a `ticker := time.NewTicker`?

Reply:
oops nevermind, I see it on line 24